input/pages/example-app-launch-symmetric-auth.md

```diff
@@ -82,14 +82,21 @@ curl -s 'https://smart.argo.run/v/r4/sim/eyJtIjoiMSIsImsiOiIxIiwiaSI6IjEiLCJqIjo
 
 ### Obtain authorization code
 
-Generate a PKCE code challenge and verifier, then redirect browser to the `authorize_endpoint` from the discovery response (newlines added for clarity):
+The client generates the following (ideally on its backend, not frontend):
```

Comment:
Recommend creating a standalone "include" file for this content so we don't accidentally get the duplicated recommendations out of sync across the different example pages where they're present.